sql/core/src/test/scala/org/apache/spark/sql/execution/SameResultSuite.scala

```diff
@@ -58,4 +61,16 @@ class SameResultSuite extends QueryTest with SharedSQLContext {
     val df4 = spark.range(10).agg(sumDistinct($"id"))
     assert(df3.queryExecution.executedPlan.sameResult(df4.queryExecution.executedPlan))
   }
+
+  test("Canonicalized result is not case-insensitive") {
+    val a = AttributeReference("A", IntegerType)()
+    val b = AttributeReference("B", IntegerType)()
+    val planUppercase = Project(Seq(a, b), LocalRelation(a))
```

Comment:
we should create valid plans...  `Project(Seq(a), LocalRelation(a, b))`

Reply:
Ok,thanks.